### ISSUE TITLE
Less: Unknown word CssSyntaxError

### ISSUE THREAD

**Issue by XhmikosR** [shellscape/postcss-less#138]
```
v10.15.3
6.4.1
stylelint@9.10.1
postcss@7.0.14
postcss-less@3.1.4
```

### Errors 

```shell
Running "stylelint:dist" (stylelint) task

<input css 1>
 51:2  ×  Unknown word   CssSyntaxError
```

### Expected Behavior

No errors to be thrown

### How can we reproduce the behavior?

I will try to pinpoint what exactly is causing this, but in case anyone wants to give it a go just checkout the `v3-dev` branch of `https://github.com/twbs/bootstrap.git`

**Comment by XhmikosR** [comment]
All right, it seems the minum less code that triggers this is:

```less
//** `<input>` background color
```

**Comment by XhmikosR** [comment]
Hmm, maybe this one is an issue with stylelint. Let me make an issue there too.

**Comment by XhmikosR** [comment]
Scratch all the above. After further looking into this it does seems it's a new regression here.

I will try to provide a patch in the next days.

**Comment by shellscape** [comment]
Thanks for reporting this. It might be due to the two asterisks in a row, possible edge case. I might have time tonight to take a look, otherwise might have to wait until Monday (will be heading out of town).

_(minor side note: please don't modify the issue template, and do please use the Edit function rather than multiple comments. I was pretty confused until I got to your last comment)_

**Comment by XhmikosR** [comment]
My main problem is that this passes your test suite. But it fails in our
branch where there are new lines and consecutive comments like the above

**Comment by shellscape** [comment]
I just ran your sample LESS and this was the result I received:

Sample:

```js
`//** Horizontal offset for forms and lists.`
```

Result:

```js
Comment {
  raws: { before: '', begin: '//', left: '', right: '' },
  type: 'comment',
  parent:
   Root {
     raws: { semicolon: false, after: '' },
     type: 'root',
     nodes: [ [Circular] ],
     source: { input: [Input], start: [Object] } },
  source:
   { start: { line: 1, column: 1 },
     input:
      Input {
        css: '//** Horizontal offset for forms and lists.',
        hasBOM: false,
        id: '<input css 1>' },
     end: { line: 1, column: 2 } },
  inline: true,
  text: '** Horizontal offset for forms and lists.' }
```

However, when I run this sample:

```js
`//** Horizontal offset for forms and lists.\n`
```

I get a `CssSyntaxError`. I'm not sure if this is what you were eluding to, but it would seem that a comment with a leading asterisk and ending with a newline causes this problem. I also confirmed this on a small sample `//*a\n`.

**Comment by XhmikosR** [comment]
Yes, it seems the newline is causing the error indeed. Because whatever I tried as one line, worked fine in your tests.

Just to make sure everything is fine, I will confirm later with our v3-dev branch.

**Comment by XhmikosR** [comment]
I tried to find a solution to this to no avail. It does throw if I do `'//** Horizontal offset for forms and lists.\n'`.

Let me know if there's something else I can do to help.
